src/Symfony/Component/Scheduler/Tests/SchedulerTest.php

```diff
@@ -21,11 +23,12 @@ class SchedulerTest extends TestCase
     public function testCanRunAndStop()
     {
         $handler = new Handler();
-        $schedule = (new Schedule())->add(RecurringMessage::every('1 millisecond', new Message()));
-        $scheduler = new Scheduler([Message::class => $handler], [$schedule]);
+        $handler->clock = $clock = new MockClock();
+        $schedule = (new Schedule())->add(RecurringMessage::every('1 second', new Message()));
+        $scheduler = new Scheduler([Message::class => $handler], [$schedule], $clock);
         $handler->scheduler = $scheduler;
 
-        $scheduler->run(['sleep' => 0]);
+        $scheduler->run(['sleep' => 1]);
```

Comment:
I expected to only have to set this to `1` with the mock clock for it to run instantly but I had add a sleep in the handler below 🤔